File: src/main/java/cookiedragon/luchadora/util/ChatUtils.java
package cookiedragon.luchadora.util;

import net.minecraft.util.text.TextComponentString;

/**
 * @author cookiedragon234 22/Dec/2019
 */
public class ChatUtils implements Globals
{
	public static void sendMessage(String message)
	{
		mc.player.sendMessage(new TextComponentString(message));
	}
}

File: src/main/kotlin/cookiedragon/luchadora/util/ChatUtils.kt
package cookiedragon.luchadora.util

import cookiedragon.luchadora.util.Globals.mc
import net.minecraft.util.text.TextComponentString

/**
 * @author cookiedragon234 17/Feb/2020
 */
object ChatUtils {
	fun printMessage(message: String) {
		mc.player.sendMessage(TextComponentString(message))
	}
}
